proto/netbookai/spawner/spawner.proto

```diff
@@ -506,4 +515,32 @@ message RegisterClusterOIDCRequest {
   string accountName = 3;
   string clusterName = 4;
 }
+
 message RegisterClusterOIDCResponse {}
+
+message Route53Record {
```

Comment:
This should have been `Route53RecordSet`

Reply:
updated